recipes/marisa-trie/meta.yaml

```diff
@@ -23,6 +23,7 @@ requirements:
   build:
     - python
     - setuptools
+    - swig  # [linux]
```

Comment:
`marisa-trie` the C++ library does indeed use SWIG for generating bindings. `pytries/marisa-trie` on the other hand does not require SWIG, so I guess we could safely remove it from the recipe.